### ISSUE TITLE
Communicate to users that we no longer support costumes/ sprites with sides > 2048 pixels

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-flash#1184]
We no longer support costumes/ sprites with sides > 2048 pixels.
We should enforce that 2048 limit at load time, or at least warn about it, since this change is confusing users.

As discussed with @cwillisf, the change (https://github.com/LLK/scratch-flash/pull/1127) that makes it so projects with oversized assets sometimes no longer work properly isn't apparent to users. https://github.com/LLK/scratch-flash/issues/1177, https://github.com/LLK/scratch-flash/pull/1127#issuecomment-232371898, and https://github.com/LLK/scratch-flash/issues/1183  seem to have been caused by https://github.com/LLK/scratch-flash/pull/1127, and each of those issues feature at least 1 confused user.

**Comment by chooper100** [comment]
Yes, this has caused many users confusion. Definitely support better communication

**Comment by jwzimmer** [comment]
@chooper100 Roger that, thank you for the feedback. : )
